Packages/com.unity.inputsystem/Documentation~/ActionsEditor.md

```diff
@@ -1,29 +1,31 @@
 ---
 uid: input-system-configuring-input
 ---
-# Configuring Input
+# Configuring Input with the Actions Editor
 
-The **Input Actions Editor** allows you to configure [Input Actions](Actions.md) and their associated [Bindings](ActionBindings.md) and [Control Schemes](ActionBindings.md#control-schemes).
+The **Input Actions Editor** allows you to edit [Action Assets](ActionAssets.md), which contain a saved configuration of [Input Actions](Actions.md) and their associated [Bindings](ActionBindings.md).
 
-Open the Input Actions settings by going to **Edit** > **Project Settings** > **Input System Package** > **Input Actions**
+It allows you to group collections of Actions into [Action Maps](ActionsEditor.html#configure-action-maps), which represent different input scenarios in your project (such as UI navigation, gameplay, etc.)
 
-![image alt text](./Images/ProjectSettingsInputActions.png)
-*The Input Actions editor in the Project Settings window*
+It also alows you to define [Control Schemes](ActionBindings.md#control-schemes) which are a way to enable or disable a set of devices, or respond to which type of device is being used. This is often useful if you want to customise your UI based on whether your users are using mouse, keyboard, or gamepad as their chosen input.
 
-The Input Action editor is divided into three panels (marked A, B & C above).
+### Action Assets and Project-Wide Actions
 
-### The default Actions
```

Comment:
Some links are broken since this rename

"ActionsEditor.html#the-default-actions in C:\build\output\Unity-Technologies\InputSystem\Temp\docstest\com.unity.inputsystem@1.8\manual\RespondingToActions.html (Tag Not Found)", "ActionsEditor.html#the-default-actions in C:\build\output\Unity-Technologies\InputSystem\Temp\docstest\com.unity.inputsystem@1.8\manual\RespondingToActions.html (Tag Not Found)",